CONTRIBUTING.md

```diff
@@ -12,7 +12,8 @@ Please note we have a code of conduct, please follow it in all your interactions
 2. Update the README.md with details of changes to the interface, this includes new environment
    variables, exposed ports, useful file locations and container parameters.
 3. Make sure that your code follows the formatting rules enforced within this repository. This can be checked by running
-`flake8 src`, assuming that you have required tools installed from `requirements-dev.txt`. Also, make sure that 
+`flake8 src`, assuming that you have required tools installed from `requirements-dev.txt`. Also, make sure that
 `mypy --show-error-codes --pretty --strict .` still runs happily against the changed code.
+4. run `pre-commit install` to set up the git hook scripts. You can also use `pre-commit run --all-files` to test locally before commiting.
```

Comment:
This can be put into one statement, as just `pre-commit` now brings everything under one command (Either as `pre-commit run --all-files` or as a pre-commit hook).
I'd advise that the mentioned mypy configuration should be put into `mypy.ini` instead.